src/components/PeopplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,138 @@
+
+        if (mother) {
+          newPerson.mother = mother;
+        }
+
+        if (father) {
+          newPerson.father = father;
+        }
+
+        return newPerson;
+      });
+
+      setPeople(peopleWithParents);
+    };
+
+    getPeopleFromServer();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      {people.length
+        ? (
+          <table
+            data-cy="peopleTable"
+            className="table is-striped is-hoverable is-narrow is-fullwidth"
+          >
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Sex</th>
+                <th>Born</th>
+                <th>Died</th>
+                <th>Mother</th>
+                <th>Father</th>
+              </tr>
+            </thead>
+            <tbody>
+              {people.map(person => {
+                const isSelected = person.slug === selectedPersonSlug;
+
+                return (
+                  <tr
+                    data-cy="person"
+                    className={classNames(
+                      { 'has-background-warning': isSelected },
+                    )}
+                  >
+                    <td>
+                      <Link
+                        to={`../${person.slug}`}
+                        className={classNames(
+                          { 'has-text-danger': person.sex === 'f' },
+                        )}
+                      >
+                        {person.name}
+                      </Link>
+                    </td>
+
+                    <td>{person.sex}</td>
+                    <td>{person.born}</td>
+                    <td>{person.died}</td>
+                    <td>
+                      {person.mother
+                        ? (
+                          <Link
+                            to={`../${person.mother?.slug}`}
+                            className={classNames(
+                              { 'has-text-danger': person.mother.sex === 'f' },
+                            )}
+                          >
+                            {person.mother?.name}
+                          </Link>
+                        )
+                        : person.motherName || '-'}
+                    </td>
+                    <td>
+                      {person.father
+                        ? (
+                          <Link
+                            to={`../${person.father?.slug}`}
+                          >
+                            {person.father?.name}
+                          </Link>
+                        )
+                        : person.fatherName || '-'}
+                    </td>
+                  </tr>
+                );
+              })}
+            </tbody>
+          </table>
+
+        )
+        : (
+          <div className="block">
+            <div className="box table-container">
+              <Loader />
+
+              {/* <p data-cy="peopleLoadingError" className="has-text-danger">
```

Comment:
This looks like unfinished logic of error handling. Maybe you forgot about it? Please make it to work or remove commented the code. A repository shouldn't have commented code.